test/boost/user_types_test.cc

```diff
@@ -628,6 +629,52 @@ SEASTAR_TEST_CASE(test_nonfrozen_user_types_prepared) {
     });
 }
 
+// This test is a boost version of sandwichBetweenUDTs test from Apache Cassandra.
```

Comment:
Since I believe you also translated the same test to cqlpy, I think this version is redundant, and it's worse (among other things, because you can't run it against Cassandra) so I think it's better to just remove this one.